### DISCUSSION TITLE
How to change the language settings and display my own text?

### DISCUSSION THREAD

**Discussion by l4ndfr05ch** [datenstrom/community#966] [Ask a question] [answered]
Hi,
I would like to change the text in the cookie window. To do this, I have changed the part of the German text in the analytics.php file (line 36). Unfortunately, the new text is not displayed. The old original text is still displayed, even on other devices and other browsers, although only the new text can be read in the analytics.php file. I am puzzled as to where the old original text is still being retrieved from. 

How can I display my own text?

Marko

**Comment by schulle4u** [comment, marked as answer]
Usually the language settings for all extensions are stored in file `system/extensions/yellow-language.ini` and have precedence over the default extension settings. Language settings in the extension's php file are only the default values and should not be changed there. 

See also: [How to change the system: Language settings](https://datenstrom.se/yellow/help/how-to-change-the-system#language-settings)

**Reply by markseuffert** [reply]
![meme-this-is-the-way](https://user-images.githubusercontent.com/2429165/136006429-2cf84dab-9707-4b0c-91f9-d9803973fecf.jpg)

**Comment by l4ndfr05ch** [comment]
Ah ok, I didn't know that the Analytics extension also creates entries in the yellow-language.ini. After I changed the cookie text there, the new text was also displayed on the website.
Thank you :-).
